### PR TITLE
Add automatic per app languages

### PR DESCRIPTION
**Pull request by Arthur-GYT**
All is in the title ^^.

https://developer.android.com/guide/topics/resources/app-languages

### CODE REVIEW

**Comment by Jean-BaptisteC** [comment]
See #5665

**Comment by Arthur-GYT** [comment]
My bad, I close this PR

**Comment by Jean-BaptisteC** [comment]
Not need to close your PR, we have just need help to finish PR and merge :)

File: android/app/src/main/res/resources.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+unqualifiedResLocale=en-US
```
Comment (biodranik) [inline android/app/src/main/res/resources.properties:1]: This line is useful, it can be merged separately.

**Pull request closed without merging**